rust-runtime/aws-smithy-http-server/examples/pokemon_service/Cargo.toml

```diff
@@ -7,7 +7,7 @@ authors = ["Smithy-rs Server Team <smithy-rs-server@amazon.com>"]
 description = "A smithy Rust service to retrieve information about Pokémon."
 
 [dependencies]
-clap = { version = "3.2.1", features = ["derive"] }
+clap = { version = "~3.2.1", features = ["derive"] }
```

Comment:
This can now be reverted as deprecations are now off by default, see clap-rs/clap#3830 for details.  When you do start working through the deprecations as part of upgrading to 4.0, we've also improved the warnings for derive users.  See clap-rs/clap#3832 for examples.